ntbs-service/appsettings.Development.json

```diff
@@ -2,14 +2,15 @@
   "AdfsOptions": {
     "Wtrealm": "https://localhost:5001/",
     "DevGroup": "Global.NIS.NTBS.NTS",
+    "AdminUserGroup": "Global.NIS.NTBS.Admin",
     "UseDummyAuth": false
   },
   "AzureAdOptions": {
     "ClientId": "<Provided by deployment secrets>",
     "ClientSecret": "<Provided by deployment secrets>",
     "Authority": "<Provided by deployment secrets>",
-    "CallbackPath": "/Index",
-    "Enabled": false
+    "CallbackPath": "/signin-oidc",
```

Comment:
Is this a real change? What does it mean?

Reply:
https://github.com/dotnet/aspnetcore/issues/26932 - this issue was the error I was getting. From the final comment on that thread it seems like it is a breaking change from migration from 2.2.